core/src/main/java/org/apache/accumulo/core/client/mapreduce/AccumuloOutputFormat.java

```diff
@@ -37,6 +38,7 @@
 import org.apache.accumulo.core.client.admin.DelegationTokenConfig;
 import org.apache.accumulo.core.client.admin.SecurityOperations;
 import org.apache.accumulo.core.client.impl.AuthenticationTokenIdentifier;
+import org.apache.accumulo.core.client.impl.ConnectionInfoImpl;
```

Comment:
Using the interface as the type should eliminate the need to depend on the impl as well.

Reply:
Fixed in 705d604759f9495cb